### DISCUSSION TITLE
Imperative mode not working?

### DISCUSSION THREAD

**Discussion by GabrielMAraujo** [ReactTooltip/react-tooltip#1225] [Q&A] [answered]
Hi,

I've been trying to do a custom implementation of the tooltip that requires me to open and close the tooltip programmatically, and it also must not close the tooltip when the mouse hovers out the anchor div.

I read in the docs that the **imperativeModeOnly** prop should suffice in this case, but I tried to use it and it seems like it's not working.

I made an example in CodeSandbox that shows this behavior (it should open the tooltip and never close it): https://codesandbox.io/p/sandbox/tooltip-forked-gf82hl
Am I doing something wrong, or is this a bug?

**Comment by gabrieljablonski** [comment, marked as answer]
It seems to be indeed a bug. See #1226 for suggested workaround until it is fixed.

**Reply by GabrielMAraujo** [reply]
Ok, thank you for your help!
